site-src/v1alpha2/concepts/api-overview.md

```diff
@@ -67,148 +68,89 @@ GatewayClass.
 
 As the Gateway spec captures user intent, it may not contain a complete
 specification for all attributes in the spec. For example, the user may omit
-fields such as addresses, TLS settings. This allows the controller
-managing the GatewayClass to provide these settings for the user, resulting in a
-more portable spec. This behaviour will be made clear using the GatewayClass
-Status object.
+fields such as addresses, TLS settings. This allows the controller managing the
+GatewayClass to provide these settings for the user, resulting in a more
+portable spec. This behaviour will be made clear using the GatewayClass Status
+object.
 
-A Gateway MAY contain one or more *Route references which serve to direct
-traffic for a subset of traffic to a specific service.*
-
-### {HTTP,TCP,Foo}Route
+### Routes
 
 Route objects define protocol-specific rules for mapping requests from a Gateway
-to Kubernetes Services.
-
-`HTTPRoute` and `TCPRoute` are currently the only defined Route objects.
-Additional protocol-specific Route objects may be added in the future.
-
-### Route binding
-
-When a Route binds to a Gateway it represents configuration that is applied on
-the Gateway that configures the underlying load balancer or proxy. How and which
-Routes bind to Gateways is controlled by the resources themselves. Route and
-Gateway resources have built-in controls to permit or constrain how they select
-each other. This is useful for enforcing organizational policies for how Routes
-are exposed and on which Gateways. Consider the following example:
-
-> A Kubernetes cluster admin has deployed a Gateway “shared-gw” in the “Infra”
-> Namespace to be used by different application teams for exposing their
-> applications outside the cluster. Teams A and B (in Namespaces “A” and “B”
-> respectively) bind their Routes to this Gateway. They are unaware of each other
-> and as long as their Route rules do not conflict with each other they can
-> continue operating in isolation. Team C has special networking needs (perhaps
-> performance, security, or criticality) and they need a dedicated Gateway to
-> proxy their application to the outside world. Team C deploys their own Gateway
-> “dedicated-gw”  in the “C” Namespace that can only be used by apps in the "C"
-> Namespace.
+to Kubernetes Services. There are 4 Route resources included with the Gateway
+API:
 
-<!-- source: https://docs.google.com/presentation/d/1neBkFDTZ__vRoDXIWvAcxk2Pb7-evdBT6ykw_frf9QQ/edit?usp=sharing -->
-![route binding](/images/gateway-route-binding.png)
+* HTTPRoute
+* TCPRoute
+* TLSRoute
+* UDPRoute
+
+Additional Route resources may be added in the future.
```

Comment:
we should filed an issue to describe dev guide for "make your own route"

Reply:
https://github.com/kubernetes-sigs/gateway-api/issues/776